ci-operator/step-registry/openshift-extended/test/supplementary/openshift-extended-test-supplementary-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "openshift-extended/test/supplementary/openshift-extended-test-supplementary-ref.yaml",
+	"owners": {
+		"approvers": [
+			"jianzhangbjz",
+			"kuiwang02"
+		],
+		"reviewers": [
+			"jianzhangbjz",
+			"kuiwang02",
+			"xia-zhao-rh"
+		]
+	}
+}
+
```

Comment:
@sunzhaohua2 need to remove the last line to resolve step-registry-metadata failire. and then we get warning here we have to accept.